frontend/src/components/Pages/Dashboard/Dashboard.jsx

```
@@ -284,138 +22,131 @@ const Dashboard = () => {
-      />
-      <div className="flex-1 p-4 sm:p-6 md:p-8 ml-0 md:ml-20 lg:ml-64 transition-all duration-300">
-        {dashboardContent}
-      </div>
+    <div className="flex bg-gray-900 min-h-screen text-white">
+      <DashboardSideNavbar isOpen={sidebarOpen} setIsOpen={setSidebarOpen} />
+      <main className={`flex-1 p-4 sm:p-6 md:p-8 ${sidebarOpen ? 'ml-64' : 'ml-0 md:ml-20'} lg:ml-64 transition-all duration-300 ease-in-out overflow-y-auto`}>
+        <button 
+          onClick={() => setSidebarOpen(!sidebarOpen)} 
+          className="md:hidden fixed top-4 left-4 z-50 p-2 bg-gray-700 rounded text-white hover:bg-gray-600 focus:outline-none focus:ring-2 focus:ring-gray-500" 
+          aria-label="Toggle sidebar"
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)